packages/falter-berlin-ssid-changer/files/hotplug.d/30-ssid-changer

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+# logger -t "ssid net/hotplug" "ssidchanger ACTION = $ACTION INTERFACE = $INTERFACE DEVICE = $DEVICE IFUPDATE_ADDRESSES = $IFUPDATE_ADDRESSES IFUPDATE_DATA = $IFUPDATE_DATA"
+
```

Comment:
It might make sense to check to see what $ACTION, $INTERFACE, and $DEVICE are and exit early

Reply:
Do you have some further thoughts on that? I don't know what exactly could be done here.

Reply:
for example, if $INTERFACE is not "ffuplink" and also not "tnl_*" then you can quickly exit.

Reply:
done in new version.

Reply:
I think you mean to put "done in new version" for a different review.  What do you think about this comment?

Reply:
No, I've made a new commit for this review.

Reply:
Now I found it.  Resolving...